zeebe/exporters/camunda-exporter/src/test/java/io/camunda/exporter/utils/SearchClientAdapter.java

```diff
@@ -114,7 +114,7 @@ public JsonNode getPolicyAsNode(final String policyName) throws IOException {
       final var request =
           Requests.builder().method("GET").endpoint("_plugins/_ism/policies/" + policyName).build();
 
-      return MAPPER.readTree(osClient.generic().execute(request).getBody().get().body());
+      return objectMapper.readTree(osClient.generic().execute(request).getBody().get().body());
```

Comment:
❓ Can we use the reader in this class as well?

Reply:
I left the objectMapper because we need both the reader and the writer in this case